packages/jest-runtime/src/transform.js

```diff
@@ -159,6 +161,18 @@ const shouldInstrument = (filename: Path, config: Config): boolean => {
   }
 
   if (
+    !config.collectCoverageOnlyFrom && // still cover if `only` is specified
+    config.collectCoverageFrom &&
+    !multimatch(
```

Comment:
that's what i was thinking about. I'll probably just take `minimatch` and join multiple globs into one regexp for easier integraiton with haste map